arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Anytrust orbit chains keyset generation'
+sidebar_label: 'Keyset Generation for Anytrust Orbits'
+description: 'Keyset generation and setup for Anytrust Orbit chains'
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Anytrust Orbit chains.'
+sidebar_position: 1
+---
+**Anytrust** chains, relies on an external Data Availability Committee (hereafter, "the Committee") to store data and provide it on demand, instead of using the parent chain as Data Availability (DA) layer. 
```

Comment:
```suggestion
**AnyTrust** chains rely on an external Data Availability Committee (DAC) to store data and provide it on demand, instead of using the parent chain as Data Availability (DA) layer. 
```

We use DAC elsewhere [[1]](https://docs.arbitrum.foundation/dao-glossary#data-availability-committee-dac); edit for consistency. This would also let us use quicklooks to expose term definitions (pulled from [Glossary CMS](https://www.notion.so/arbitrum/Glossary-CMS-0718756f84b44dcfa8907a72854550cf)) on hover.